ev3dev2/wheel.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+"""
+Wheel and Rim classes
+
+A great reference when adding new wheels is http://wheels.sariel.pl/
+"""
+from math import pi
+
+
+class Wheel(object):
+    """
+    A base class for various types of wheels, tires, etc
+    All units are in mm
+    """
+
+    def __init__(self, diameter_mm, width_mm):
+        self.diameter_mm = float(diameter_mm)
+        self.width_mm = float(width_mm)
+        self.radius_mm = float(diameter_mm / 2)
+        self.circumference_mm = diameter_mm * pi
+
+
+class EV3RubberWheel(Wheel):
```

Comment:
I think we should add some other commonly used wheels but will do that via another pull request

Reply:
Are rubber wheels from home edition and from education edition have the same diameter and width?

Reply:
I recall there being different rubber tires and associated rims which looked similar but had different diameters. I think there was one inner rim with three peg holes for a special axle mount piece and a different one which was a bit smaller and lacked them. So maybe we need particular part numbers here.